roles/template_service_broker/tasks/install.yml

```diff
@@ -42,6 +43,28 @@
     src: openshift-ansible-catalog-console.js
     dest: /etc/origin/master/openshift-ansible-catalog-console.js
 
+# Check that the TSB is running
+- name: Verify that TSB is running
+  command: >
+    curl -k https://apiserver.openshift-template-service-broker.svc/healthz
+  args:
+    # Disables the following warning:
+    # Consider using get_url or uri module rather than running curl
+    warn: no
+  register: api_health
+  until: api_health.stdout == 'ok'
+  retries: 120
+  delay: 1
+  changed_when: false
+
+- set_fact:
+    __ca_bundle: "blahblahblah"
```

Comment:
it's the ca bundle for the cluster (the same one it uses to create service signing certs).  not sure exactly where you can get that from though.  Presumably we're generating it during the install somewhere, it's going to live on the master.